### ISSUE TITLE
Ownership transfer

### ISSUE THREAD

**Issue by fiLLLip** [fiLLLip/namron-homey#26]
Hi all,

today I was contacted by Namron regarding this app and its future. Namron themselves want to continue development of this app and add support for their entire product range. This is something I applaud, and really look forward to see. I just transferred the ownership of the app on Homey developer portal to Namron, so the last update to it from me have been published.

I am not sure if this repo will receive any updates, or how they will move forward. The only thing I now know is they want to continue development and focus on the upcoming Homey release (Homey v5 with App SDK 3).

As for stats, this project actually had 614 live users as of today when I transferred the app. That is pretty cool, thanks :)

**Comment by aasmundbo** [comment]
Thank you for the great work, @fiLLLip !

Hopefully Namron taking over will mean more devices get supported and more features get added!

Do you know how/where to request new features and bugs to Namron?

**Comment by fiLLLip** [comment]
I have an ongoing communication with them about the future of this repo, so hopefully I will get some answers soon. They will probably update the application on Homey I guess, with the correct links to support/issues. I will keep you posted :sunglasses:

**Comment by sevrob** [comment]
Thanx for the great work @fiLLLip . You were the reason I ended up with Namron stuff!

**Comment by espenjohannessen** [comment]
Did you get any more information from Namron about this project moving forward @fiLLLip ? As I wrote in #29 they've recently replied to reviews on the Homey site that an updated app will be released with Homey v5, but it would be great to get something working in the meantime or at least an ETA. I just blew like 15K on smart home stuff and if the updated app is far from beeing ready I'd be better off returning some of this stuff.

**Comment by fiLLLip** [comment]
Nope, no info on the roadmap, just that they would take over the development. I asked if they wanted ownership of this repo as well, but no updates after that question.
